src/HDInsight/HDInsight/Models/Management/AzureHdInsightManagementClient.cs

```diff
@@ -225,5 +226,21 @@ public virtual void RestartHosts(string resourceGroupName, string clusterName, I
         {
             HdInsightManagementClient.VirtualMachines.RestartHosts(resourceGroupName, clusterName, hosts);
         }
+        
+        private void ResetClusterIdentity(ClusterCreateParametersExtended createParams, string cloudAadAuthority, string cloudDataLakeAudience)
```

Comment:
Nit - I would drop the word cloud. E.g `cloudAadAuthority` -> `aadAuthority`

Reply:
> Nit - I would drop the word cloud. E.g `cloudAadAuthority` -> `aadAuthority`

Changed.